src/libserialize/serialize.rs

```diff
@@ -911,4 +911,5 @@ impl<T: UseSpecializedDecodable> Decodable for T {
 impl<'a, T: ?Sized + Encodable> UseSpecializedEncodable for &'a T {}
 impl<T: ?Sized + Encodable> UseSpecializedEncodable for Box<T> {}
 impl<T: Decodable> UseSpecializedDecodable for Box<T> {}
+impl<'a, T: ?Sized + Decodable> UseSpecializedDecodable for &'a T {}
 impl<'a, T: ?Sized + Decodable> UseSpecializedDecodable for &'a [T] {}
```

Comment:
I think this can be removed now that the general `&'a T` impl exists.

Reply:
That doesn't seem to work, seems like due to a specialization bug. See the travis failure.